student/tests.py

```diff
@@ -1,15 +1,119 @@
 import pytest
+from django.contrib.auth.models import User
+from student.models import Student, EducationalInstitution
+from datetime import date
 
 
-def test_basic_math_operation():
-    assert 10 == 2 * 5
+@pytest.fixture
+def create_user():
+    user = User.objects.create_user("test", "testpassword")
+    user.save()
+    return user
 
 
 @pytest.fixture
-def numbers_list():
-    return [1, 2, 3, 4]
+def create_institution():
+    institution = EducationalInstitution(name="A")
+    institution.save()
+    return institution
+
+
+@pytest.fixture
+def create_student(create_user, create_institution):
+    student = Student(user=create_user, full_name="test user", email="test@test.com", date_of_birth=date(2020, 7, 13),
+                      phone_number="0000000000", educational_institution=create_institution,
+                      graduation_date=date(2020, 7, 13))
+    student.save()
+    return student
+
+
+@pytest.mark.django_db
+def test_get_institutions():
```

Comment:
test names should indicate what you are testing, don't be shy about making them long! For example:

`test_get_institutions_returns_all_institutions_as_a_query_set`

Reply:
OK